scripts/make_stm32_module.py

```diff
@@ -18,24 +20,25 @@ def module(make_serie):
 
     make_path = os.getcwd()
 
-    if 'stm32' not in make_serie:
-        make_serie = 'stm32' + make_serie    # example 'stm32f3'
+    if "stm32" not in make_serie:
+        make_serie = "stm32" + make_serie  # example 'stm32f3'
 
-    make_seriexx = make_serie.lower() + "xx"    # example 'stm32f3xx'
-    make_SERIE = make_serie.upper()    # example 'STM32F3'
-    make_SERIExx = make_SERIE + "xx"    # example 'STM32F3xx'
+    make_seriexx = make_serie.lower() + "xx"  # example 'stm32f3xx'
+    make_SERIE = make_serie.upper()  # example 'STM32F3'
+    make_SERIExx = make_SERIE + "xx"  # example 'STM32F3xx'
```

Comment:
PEP8 variable naming should be followed https://pep8.org/#naming-conventions

Reply:
Done